osc/osc.go

```diff
@@ -968,20 +964,30 @@ func readPaddedString(reader *bufio.Reader) (string, int, error) {
 		}
 	}
 
-	return str, n, nil
+	// Strip off the string delimiter
+	return str[:len(str)-1], n, nil
 }
 
 // writePaddedString writes a string with padding bytes to the a buffer.
 // Returns, the number of written bytes and an error if any.
 func writePaddedString(str string, buf *bytes.Buffer) (int, error) {
+	// Truncate at the first null, just in case there is more than one present
+	nullIndex := strings.Index(str, "\x00")
+	if (nullIndex > 0) {
```

Comment:
Gofmt ist complaining about the parenthesis. Therefore the build fails.

Reply:
Done